### PR TITLE
[RM Appendix - A] Renaming the chapter

### LINKED ISSUE #1486: [WSRM Ch1-Ch6] VIM, SW and HW is still mentioned in RM

### PR DESCRIPTION
**Pull request by EditKoselak**
Partially closes #1486

### CODE REVIEW

**Comment by EditKoselak** [comment]
> 
> 
> This chapter title is also referred to by name in RM Chapter01.
> https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter01.md
> 
> For consistency, can you please change it in both places?

Thank you, we have corrected it in RM Chapter01: #1487, and in the README: #1540

File: doc/ref_model/chapters/appendix-a.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [<< Back](../../ref_model)
-# Appendix A - Guidelines for VNF Vendors
+# Appendix A - Guidelines For Application Vendors
 <p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
```
Comment (ASawwaf) [inline doc/ref_model/chapters/appendix-a.md:3]: @kedmison we are using Workload not application, what did you think

Comment (kedmison) [reply doc/ref_model/chapters/appendix-a.md:3]: We have a lot of mixed use of VNF, Workload, and Application.  I think we generally perceive workload and application as consistent with each other, but VNF is perhaps a bit too specific relative to VM-based virtualization.  
So, I think it's acceptable.

**Comment by rabi-abdel** [comment]
@ASawwaf can you re-review/approve please

**Pull request merged**